src/patternfly/components/AboutModalBox/about-modal-box.scss

```diff
@@ -1,14 +1,12 @@
 // @debug $about-modal-box; // check your variable names located in src/patternfly/sass-utilities/component-namespaces.scss
 
-.#{$about-modal-box} {
+:root {
   // Component variables
   --#{$about-modal-box}--BackgroundImage: none;
-  --#{$about-modal-box}--BackgroundColor: var(--#{$pf-global}--palette--black-1000); // Modal uses a non-standard background color
+  --#{$about-modal-box}--BackgroundColor: var(--pf-t--global--background--color--floating--default);
```

Comment:
We could also remove this if the intent from design is that this comes from the modal. Similar to the rounded corners and box-shadow. The design would even be a lot more clear if the about modal were created inside of the modal component, so it's clear what parts of the design are from what components.

Reply:
It wasn't clear to me whether this background color would simply be the modal background color or if them being the same was a coicidence. @lboehling ?